classes/notification/NotificationSubscriptionSettingsDAO.inc.php

```diff
@@ -16,13 +16,15 @@
  *  delivered to them.
  */
 
+define('BLOCKED_NOTIFICATION_KEY', 'blocked_notification');
```

Comment:
I would actually prefer us to start putting constants into classes as we start using modern namespaces/segmentation/autoloading and move away from global-ish habits. Is there a reason for moving these out of the class?

Reply:
Fixed.

Used the `NotificationSubscriptionSettingsDAO::BLOCKED_NOTIFICATION_KEY` and `NotificationSubscriptionSettingsDAO::BLOCKED_EMAIL_NOTIFICATION_KEY` notation.